README.md

```diff
@@ -94,7 +94,42 @@ To build the project execute the following command:
   ./gradlew build
 ```
 
-## To use ccd-docker elements
+## Minimal docker development environment
+
+##### 1) Install jq
+
+```bash
+sudo apt-get install jq
+```
+
+For mac 
+```bash
+brew install jq
+```
+
+##### 2) Login to azure
+
+```bash
+az login
+az acr login --name hmctspublic --subscription DCD-CNP-Prod
```

Comment:
we may consider combining the below command to one ,something like-
az acr login --name hmctspublic && az acr login --name hmctsprivate